localdev/docker-compose.yml

```diff
@@ -1,14 +1,15 @@
-version: "3.3"
+version: "3.4"
 services:
   mysqldb:
     environment:
       - MYSQL_ROOT_PASSWORD=root123
       - MYSQL_DATABASE=trinogateway
-    image: mysql:5.5
+    image: mysql:5.7
```

Comment:
If a newer mysql version works I would much rather upgrade more

Reply:
I'll check to see with 8.x versions. 
5.7 was just mere patch for m1 version to work.

Reply:
I just tried with 8.0 and there is issue with index size in [exact_match_source_selectors](https://github.com/trinodb/trino-gateway/blob/main/gateway-ha/src/main/resources/gateway-ha-persistence.sql#L67-L77) table's unique index, as it is innodb with utf8mb4 (4 bytes)

So, to resolve the issue, we need to either
1. decrease size of the variables: resource_group_id(256), source(512), environment(128), query_type(128) from 1024 -> 768 (-256).
2. Or use utf8(mb3) instead of mb4.

Reply:
It sounds like a more complex migration to MySQL 8 - maybe that should be a separate PR? If we change the schema then we should provide a migration as well.

Reply:
I gree separating PR would be better. I noticed upgrading to mysql 8 may cause other issues.

FYI. trino solves this issue by giving limits to each unique key index `UNIQUE (source(128), environment, query_type(128), resource_group_id)` but when I tried to do this in gateway
there is syntax issue when activejdbc executes sql. 
- https://github.com/trinodb/trino/blob/master/plugin/trino-resource-group-managers/src/main/resources/db/migration/mysql/V4__add_exact_match_source_selectors.sql#L8

Reply:
Yea .. lets stick to 5 for now

Reply:
Sure i just created an issue to be handled in future
- https://github.com/trinodb/trino-gateway/issues/54